pkgs/development/python-modules/treex/default.nix

```diff
@@ -31,15 +34,16 @@ buildPythonPackage rec {
   # At the time of writing (2022-03-29), rich is currently at version 11.0.0.
   # The treeo dependency is compatible with a patch, but not marked as such in
   # treex. See https://github.com/cgarciae/treex/issues/68.
-  postPatch = ''
-    substituteInPlace pyproject.toml \
-      --replace 'rich = "^11.2.0"' 'rich = "*"' \
-      --replace 'treeo = "^0.0.10"' 'treeo = "*"' \
-      --replace 'certifi = "^2021.10.8"' 'certifi = "*"'
-  '';
+  pythonRelaxDeps = [
```

Comment:
Cool, didn't know about this! Are there docs somewhere?

Reply:
I found this one https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md :)